frontend/webpack.config.prod.js

```diff
@@ -3,6 +3,12 @@ const { resolve } = require("path");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const CopyPlugin = require("copy-webpack-plugin");
 const CompressionPlugin = require("compression-webpack-plugin");
+const webpack = require("webpack");
+const dotenvExpand = require("dotenv-expand");
+const dotenv = require("dotenv");
+
+const myEnv = dotenv.config();
+dotenvExpand(myEnv);
```

Comment:
Same changes as above

Reply:
@jyash97 process is not defined .. to do so we need to do so

Reply:
we are just defining a new variable using define plugin in this case `process.env` will point to env variables. I tried it locally it worked for me.